src/components/index.ts

```diff
@@ -1,28 +1,28 @@
-export { default as VideoGallery } from './VideoGallery'
-export { default as CoverVideo } from './CoverVideo'
-export { default as ChannelGallery } from './ChannelGallery'
-export { default as ViewerTopbar } from './Topbar/ViewerTopbar'
-export { default as PublishingTopbar } from './Topbar/StudioTopbar'
-export { default as VideoGrid } from './VideoGrid'
-export { default as PlaceholderVideoGrid } from './PlaceholderVideoGrid'
-export { default as VideoPreview, VideoPreviewPublisher } from './VideoPreview'
+export { VideoGallery } from './VideoGallery'
```

Comment:
Let's use `export * from` for all the files here unless there's a good reason not to